templates/expo-template-bare-minimum/ios/Podfile

```diff
@@ -1,15 +1,20 @@
+require File.join(File.dirname(`node --print "require.resolve('expo/package.json')"`), "scripts/autolinking")
 require File.join(File.dirname(`node --print "require.resolve('react-native/package.json')"`), "scripts/react_native_pods")
-require File.join(File.dirname(`node --print "require.resolve('react-native-unimodules/package.json')"`), "cocoapods.rb")
-require File.join(File.dirname(`node --print "require.resolve('expo-modules-core/package.json')"`), "scripts/autolinking")
 require File.join(File.dirname(`node --print "require.resolve('@react-native-community/cli-platform-ios/package.json')"`), "native_modules")
 
-platform :ios, '11.0'
+platform :ios, '12.0'
+
+require 'json'
+podfile_properties = JSON.parse(File.read('./Podfile.properties.json')) rescue {}
```

Comment:
It might be better to have this in `package.json` — what do you think?

Reply:
once thought of this. but config-plugin may change the target (in this case, package.json) during expo-cli prebuilding and makes git in dirty state. a separated `ios/Podfile.properties.json` could prevent this case. user who want to keep managed workflow could also safely remove entire ios/ folder.